binn/binn.py

```diff
@@ -1,17 +1,17 @@
 import collections
 
 import torch
-from pytorch_lightning import LightningModule
+import lightning as L
```

Comment:
Can this be changed to ```import lightning.pytorch as pl```, that is still the convention in their docs, and avoids the capital ```L```, which we should not use

Reply:
Done